plugin/webhook/webhook.go

```diff
@@ -121,14 +117,9 @@ func (c *Context) getMetaList() []meta {
 			Value: c.TaskResult.Status,
 		})
 		if c.TaskResult.Detail != "" {
-			resultDetailRune := []rune(c.TaskResult.Detail)
-			if len(resultDetailRune) > 200 {
-				resultDetailRune = resultDetailRune[:200]
-				resultDetailRune = append(resultDetailRune, []rune("... (view details in Bytebase)")...)
-			}
 			m = append(m, meta{
 				Name:  "Result Detail",
-				Value: string(resultDetailRune),
+				Value: common.TruncateStringWithDescription(c.TaskResult.Detail, 200),
```

Comment:
Is there any reason that we use different limit size for webhook and feishu?

Reply:
I found 450 for feishu approval card is okay, and whoever set the 200 limit might also find it okay. The limit is arbitrary.

Reply:
Let's be consistent if possible for consistent Feishu experience.

Reply:
Webhook uses message cards, while external approval uses feishu approvals.
These are two different systems/experiences IMO, so it is consistent.